src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -219,7 +224,17 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
   const getInputComponent = (): JSX.Element => {
     return selectedIdType.label === "Passport" &&
       selectedIdType.scannerType === "NONE" ? (
-      <ManualPassportInput
+      <PassportInputSection
```

Comment:
Can change this to be `InputPassportSection` to be similar to `InputIdSection` 🤔